src/AppBundle/Entity/Report/Report.php

```diff
@@ -712,21 +721,41 @@ public function getStatus()
     }
 
     /**
-     * @return ArrayCollection
+     * @return ArrayCollection|Document[]
      */
     public function getDocuments()
     {
         return $this->documents;
     }
 
     /**
-     * @param ArrayCollection $documents
-     *
-     * @return $this
+     * @param Document $document
      */
-    public function setDocuments($documents)
+    public function addDocument(Document $document)
     {
-        $this->documents = $documents;
-        return $this;
+        if (!$this->documents->contains($document)) {
+            $this->documents->add($document);
+        }
     }
```

Comment:
No thats fine. I don't believe in creating methods just for the sake of it. Hence my question whether JMS relies on them (I think Zend might)